serial_communication/web/app.py

```diff
@@ -38,10 +38,10 @@ def get_serial_ports():
 
 
 # Replace with the default serial port
-ser = update_serial_port(TTGO_TCALL_PORT)
-set_serial_object(ser)
-if COMMUNICATION_PORT is None:
-    COMMUNICATION_PORT = ser
+# ser = update_serial_port(TTGO_TCALL_PORT)
+# set_serial_object(ser)
+# if COMMUNICATION_PORT is None:
+#     COMMUNICATION_PORT = ser
```

Comment:
Please remove these comments in future commits and try not to commit code with comments.

Reply:
sure